src/js/components/NameValueList/__tests__/NameValueList-test.tsx

```diff
@@ -113,6 +113,42 @@ describe('NameValueList', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  test(`should render correct width with max and min`, () => {
+    const { asFragment } = render(
+      <Grommet>
+        <NameValueList
+          nameProps={{ width: { min: 'xsmall', max: 'max-content' } }}
+        >
+          {Object.entries(data).map(([name, value]) => (
+            <NameValuePair key={name} name={name}>
+              {value}
+            </NameValuePair>
+          ))}
+        </NameValueList>
+      </Grommet>,
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test(`should render correct width with max and min`, () => {
+    const { asFragment } = render(
+      <Grommet>
+        <NameValueList
+          valueProps={{ width: { min: 'xsmall', max: 'max-content' } }}
```

Comment:
Changing up the values a bit.

```suggestion
          valueProps={{ width: { min: 'small', max: 'medium' } }}
```